src/app/sidebar/Sidebar.tsx

```diff
@@ -31,24 +30,33 @@ export function Sidebar({ term }: SidebarProps): JSX.Element {
       return dict;
     }, {} as { [subject: string]: Course[] }) ?? {};
 
+  const handleClick = useCallback(
+    (pid: string) => {
+      setPid && setPid(pid);
+    },
+    [setPid]
+  );
+
   return (
-    <Flex id="scrollableFlex" maxH="100vh" bg="#E4E4E4" p="4" overflow="auto">
-      <InfiniteScroll
-        dataLength={subjects?.length ?? 0}
-        next={() => {}}
-        hasMore={false}
-        loader={<Heading size="sm">Loading...</Heading>}
-        scrollableTarget="scrollableFlex"
-        endMessage={<div />}
-      >
+    <Box>
+      <Box>{/* <Heading>Search</Heading> */}</Box>
+      <Flex id="scrollableFlex" maxH="100vh" bg="#E4E4E4" p="2" overflow="auto" direction="column">
         {subjects?.map((subject, index) => (
           <CardDropDown key={index} subject={subject.subject} title={subject.title}>
             {parsedCourses[subject.subject]?.map((course, index) => (
-              <Card key={index} title={course.title} code={course.code} subject={course.subject} />
+              <Card
```

Comment:
kinda unrelated to this PR, but can we make course cards behave like buttons on hover? i.e. have the cursor turn to a pointer